src/components/Pagination/styled.js

```diff
@@ -5,23 +5,35 @@ export const Wrapper = styled.div`
 
   display: flex;
   justify-content: center;
+`;
 
-  & a {
-    color: black;
-    float: left;
-    padding: 8px 16px;
-    text-decoration: none;
-    transition: background-color 0.3s;
-    border: 1px solid #ddd;
-  }
+export const Link = styled.div`
```

Comment:
Why is this called `Link` if the wrapper is a `div` 🤔 ?. This can lead to confusions now that you have added React Router (it has the Link import as well). I'd suggest you to change the name, and instead of `div` change it to `a`.